docs-en/chapter_data_structure/character_encoding.md

```diff
@@ -1,36 +1,36 @@
 # Character Encoding *
 
-In computers, all data is stored in binary form, and the character `char` is no exception. To represent characters, we need to establish a "character set" that defines a one-to-one correspondence between each character and binary numbers. With a character set, computers can convert binary numbers to characters by looking up a table.
+In the computer system, all data is stored in binary form, including characters (char). To represent characters, we need to develop a "character set" that defines a one-to-one mapping between each character and binary numbers. With the character set, computers can convert binary numbers to characters by looking up the table.
```

Comment:
I think to keep the original but by changing it a little.
"all data is stored in binary form, including characters (char)."->"all data is stored in binary form, and characters (represented by char) are no exception."

this is optional:
"With the character set, computers can convert binary numbers to characters by looking up the table." ->
"Computers utilize this character set to convert binary numbers to characters through table lookup."